Code/Engine/RendererCore/Debug/DebugTextComponent.h

```diff
@@ -32,6 +32,8 @@ class EZ_RENDERERCORE_DLL ezDebugTextComponent : public ezComponent
   float m_fValue2 = 0.0f;                                         // [ property ]
   float m_fValue3 = 0.0f;                                         // [ property ]
 
+  float m_fMaxDistance = 100.0f;                                  // [ property ]
```

Comment:
100m is already pretty far way. The default could probably be significantly lower (somewhere between 10 and 25 meters maybe).

Reply:
I chose 100 to not break existing behavior too much, so people don't wonder where there debug text went.

Reply:
Alright, reduced it to 10 and added some fading.